API.Benchmark/ArchiveServiceBenchmark.cs

```diff
@@ -24,16 +24,16 @@ public class ArchiveServiceBenchmark
     private readonly ArchiveService _archiveService;
     private readonly IDirectoryService _directoryService;
     private readonly IImageService _imageService;
-    private readonly PngEncoder _pngEncoder = new PngEncoder();
-    private readonly WebpEncoder _webPEncoder = new WebpEncoder();
+    private readonly IImageFactory _imageFactory;
     private const string SourceImage = "C:/Users/josep/Pictures/obey_by_grrsa-d6llkaa_colored_by_me.png";
```

Comment:
We should probably remove this hardcoded path and point to an arbitrary image in TestData

Reply:
Done